sensors/sensors_test.go

```diff
@@ -187,6 +187,8 @@ func TestTimedLidarSensorReading(t *testing.T) {
 
 	t.Run("when a live lidar succeeds, returns current time in UTC and the reading", func(t *testing.T) {
 		beforeReading := time.Now().UTC()
+		time.Sleep(10 * time.Millisecond)
```

Comment:
[note] wait was added as flaky tests would occur if execution is so fast that `tsr.ReadingTime `and `beforeReading` are equal